Humidity Control not updating LCD or Pin

For EU/Generic systems, humidity control and hot water have the following same issue:

Web UI shows correctly the state as ON
LCD UI shows incorrectly the state as OFF (icon stays white)
Pin and relay is OFF

> Can you post events.log between the times where openhab.log shows "Initializing settings" and "Initializing settings" plus a second (just in case).

events.log:
```
2020-02-18 12:10:42.641 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.6
2020-02-18 12:10:42.659 [vent.ItemStateChangedEvent] - MyHumiProxy changed from NULL to 36.6
2020-02-18 12:10:42.684 [ome.event.ItemCommandEvent] - Item 'SystemType' received command EU
2020-02-18 12:10:42.692 [nt.ItemStatePredictedEvent] - SystemType predicted to become EU
2020-02-18 12:10:42.826 [ome.event.ItemCommandEvent] - Item 'TempUnit' received command F
2020-02-18 12:10:42.834 [nt.ItemStatePredictedEvent] - TempUnit predicted to become F
2020-02-18 12:10:44.642 [vent.ItemStateChangedEvent] - MyHumi changed from 36.6 to 36.7
2020-02-18 12:10:54.821 [vent.ItemStateChangedEvent] - MyHumi changed from 36.7 to 36.8
2020-02-18 12:10:59.312 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.8
2020-02-18 12:10:59.326 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:10:59.349 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.8
2020-02-18 12:10:59.354 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:10:59.367 [vent.ItemStateChangedEvent] - MyHumiProxy changed from 36.6 to 36.8
2020-02-18 12:10:59.373 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.8
2020-02-18 12:10:59.380 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.8
2020-02-18 12:10:59.408 [ome.event.ItemCommandEvent] - Item 'MainSwitch' received command ON
2020-02-18 12:10:59.656 [ome.event.ItemCommandEvent] - Item 'HeatingMode' received command ON
2020-02-18 12:10:59.693 [ome.event.ItemCommandEvent] - Item 'HotWaterMode' received command ON
2020-02-18 12:10:59.955 [ome.event.ItemCommandEvent] - Item 'HumidityMode' received command ON
2020-02-18 12:10:59.966 [ome.event.ItemCommandEvent] - Item 'Heating2' received command OFF
2020-02-18 12:10:59.981 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:10:59.989 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:11:00.028 [ome.event.ItemCommandEvent] - Item 'TempSetpoint' received command 90
2020-02-18 12:11:00.060 [ome.event.ItemCommandEvent] - Item 'TempSetpointF' received command 90
2020-02-18 12:11:00.090 [ome.event.ItemCommandEvent] - Item 'HumiSetpoint' received command 25
2020-02-18 12:11:00.107 [ome.event.ItemCommandEvent] - Item 'HeatingBoostTime' received command 10
2020-02-18 12:11:00.129 [ome.event.ItemCommandEvent] - Item 'HotWaterBoostTime' received command 10
2020-02-18 12:11:00.184 [ome.event.ItemCommandEvent] - Item 'HumiBoostTime' received command 10
2020-02-18 12:11:00.191 [ome.event.ItemCommandEvent] - Item 'PreviousTempReading' received command 73
2020-02-18 12:11:00.215 [ome.event.ItemCommandEvent] - Item 'PreviousHumiReading' received command 37.0
2020-02-18 12:11:00.223 [ome.event.ItemCommandEvent] - Item 'Heating2Time' received command 3
2020-02-18 12:11:00.236 [ome.event.ItemCommandEvent] - Item 'Heating2Delta' received command 2
2020-02-18 12:11:00.422 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_IP' received command 10.10.10.52
2020-02-18 12:11:00.427 [nt.ItemStatePredictedEvent] - Network_WLAN_IP predicted to become 10.10.10.52
2020-02-18 12:11:00.480 [ome.event.ItemCommandEvent] - Item 'Network_SSID' received command BOB
2020-02-18 12:11:00.497 [nt.ItemStatePredictedEvent] - Network_SSID predicted to become xxx
2020-02-18 12:11:00.598 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_INFO' received command 92
2020-02-18 12:11:00.625 [nt.ItemStatePredictedEvent] - Network_WLAN_INFO predicted to become 92
2020-02-18 12:11:00.630 [vent.ItemStateChangedEvent] - Network_WLAN_INFO changed from 86 to 92
2020-02-18 12:11:00.667 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_MAC' received command b8:27:eb:xx:xx:43
2020-02-18 12:11:00.684 [nt.ItemStatePredictedEvent] - Network_WLAN_MAC predicted to become b8:27:eb:xx:xx:43
2020-02-18 12:11:00.697 [vent.ItemStateChangedEvent] - Network_WLAN_MAC changed from NULL to b8:27:eb:xx:xx:43
2020-02-18 12:11:00.773 [ome.event.ItemCommandEvent] - Item 'System_CPU_TEMP' received command 122 °F
2020-02-18 12:11:00.788 [nt.ItemStatePredictedEvent] - System_CPU_TEMP predicted to become 122 °F
2020-02-18 12:11:01.859 [ome.event.ItemCommandEvent] - Item 'System_CPU_LOAD' received command 37
2020-02-18 12:11:01.866 [nt.ItemStatePredictedEvent] - System_CPU_LOAD predicted to become 37
2020-02-18 12:11:01.870 [vent.ItemStateChangedEvent] - System_CPU_LOAD changed from 49 to 37
2020-02-18 12:11:04.991 [vent.ItemStateChangedEvent] - MyHumi changed from 36.8 to 36.9
2020-02-18 12:11:19.044 [ome.event.ItemCommandEvent] - Item 'SystemType' received command EU
2020-02-18 12:11:19.049 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.9
2020-02-18 12:11:19.060 [nt.ItemStatePredictedEvent] - SystemType predicted to become EU
2020-02-18 12:11:19.069 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.9
2020-02-18 12:11:19.082 [ome.event.ItemCommandEvent] - Item 'TempUnit' received command F
2020-02-18 12:11:19.085 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:11:19.092 [vent.ItemStateChangedEvent] - MyHumiProxy changed from 36.8 to 36.9
2020-02-18 12:11:19.099 [nt.ItemStatePredictedEvent] - TempUnit predicted to become F
2020-02-18 12:11:19.105 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:11:19.126 [ome.event.ItemCommandEvent] - Item 'MainSwitch' received command ON
2020-02-18 12:11:19.145 [ome.event.ItemCommandEvent] - Item 'HeatingMode' received command ON
2020-02-18 12:11:19.164 [ome.event.ItemCommandEvent] - Item 'HotWaterMode' received command ON
2020-02-18 12:11:19.178 [ome.event.ItemCommandEvent] - Item 'HumidityMode' received command ON
2020-02-18 12:11:19.201 [ome.event.ItemCommandEvent] - Item 'Heating2' received command OFF
2020-02-18 12:11:19.232 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:11:19.237 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:11:19.275 [ome.event.ItemCommandEvent] - Item 'TempSetpoint' received command 90
2020-02-18 12:11:19.306 [ome.event.ItemCommandEvent] - Item 'TempSetpointF' received command 90
2020-02-18 12:11:19.325 [ome.event.ItemCommandEvent] - Item 'HumiSetpoint' received command 25
2020-02-18 12:11:19.353 [ome.event.ItemCommandEvent] - Item 'HeatingBoostTime' received command 10
2020-02-18 12:11:19.368 [ome.event.ItemCommandEvent] - Item 'HotWaterBoostTime' received command 10
2020-02-18 12:11:19.379 [ome.event.ItemCommandEvent] - Item 'HumiBoostTime' received command 10
2020-02-18 12:11:19.406 [ome.event.ItemCommandEvent] - Item 'PreviousTempReading' received command 73
2020-02-18 12:11:19.413 [ome.event.ItemCommandEvent] - Item 'PreviousHumiReading' received command 37.0
2020-02-18 12:11:19.433 [ome.event.ItemCommandEvent] - Item 'Heating2Time' received command 3
2020-02-18 12:11:19.455 [ome.event.ItemCommandEvent] - Item 'Heating2Delta' received command 2
2020-02-18 12:11:19.503 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_IP' received command 10.10.10.52
2020-02-18 12:11:19.512 [nt.ItemStatePredictedEvent] - Network_WLAN_IP predicted to become 10.10.10.52
2020-02-18 12:11:19.552 [ome.event.ItemCommandEvent] - Item 'Network_SSID' received command BOB
2020-02-18 12:11:19.559 [nt.ItemStatePredictedEvent] - Network_SSID predicted to become BOB
2020-02-18 12:11:19.609 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_INFO' received command 90
2020-02-18 12:11:19.614 [nt.ItemStatePredictedEvent] - Network_WLAN_INFO predicted to become 90
2020-02-18 12:11:19.703 [vent.ItemStateChangedEvent] - Network_WLAN_INFO changed from 92 to 90
2020-02-18 12:11:19.745 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_MAC' received command b8:27:eb:3d:82:43
2020-02-18 12:11:19.750 [nt.ItemStatePredictedEvent] - Network_WLAN_MAC predicted to become b8:27:eb:3d:82:43
2020-02-18 12:11:19.798 [ome.event.ItemCommandEvent] - Item 'System_CPU_TEMP' received command 122 °F
2020-02-18 12:11:19.802 [nt.ItemStatePredictedEvent] - System_CPU_TEMP predicted to become 122 °F
2020-02-18 12:11:20.859 [ome.event.ItemCommandEvent] - Item 'System_CPU_LOAD' received command 0
2020-02-18 12:11:20.865 [nt.ItemStatePredictedEvent] - System_CPU_LOAD predicted to become 0
2020-02-18 12:11:20.886 [vent.ItemStateChangedEvent] - System_CPU_LOAD changed from 37 to 0
2020-02-18 12:11:20.920 [ome.event.ItemCommandEvent] - Item 'System_Used_Space' received command 45 %
2020-02-18 12:11:20.928 [nt.ItemStatePredictedEvent] - System_Used_Space predicted to become 45 %
2020-02-18 12:11:21.018 [ome.event.ItemCommandEvent] - Item 'MyTempProxy' received command 73
2020-02-18 12:11:21.021 [nt.ItemStatePredictedEvent] - MyTempProxy predicted to become 73
2020-02-18 12:11:21.041 [ome.event.ItemCommandEvent] - Item 'PreviousTempReading' received command 0
2020-02-18 12:11:21.045 [vent.ItemStateChangedEvent] - MyTempProxy changed from NULL to 73
2020-02-18 12:11:21.048 [vent.ItemStateChangedEvent] - PreviousTempReading changed from 73 to 0
2020-02-18 12:11:21.065 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.9
2020-02-18 12:11:21.070 [vent.ItemStateChangedEvent] - MyTempProxyF changed from NULL to 73
2020-02-18 12:11:21.078 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.9
2020-02-18 12:11:21.094 [ome.event.ItemCommandEvent] - Item 'PreviousHumiReading' received command 0
2020-02-18 12:11:21.098 [ome.event.ItemCommandEvent] - Item 'MyTempProxy' received command 73
2020-02-18 12:11:21.111 [nt.ItemStatePredictedEvent] - MyTempProxy predicted to become 73
2020-02-18 12:11:21.117 [ome.event.ItemCommandEvent] - Item 'HeatingPin' received command ON
2020-02-18 12:11:21.121 [vent.ItemStateChangedEvent] - PreviousHumiReading changed from 37.0 to 0
2020-02-18 12:11:21.139 [nt.ItemStatePredictedEvent] - HeatingPin predicted to become ON
2020-02-18 12:11:21.148 [vent.ItemStateChangedEvent] - HeatingPin changed from NULL to ON
2020-02-18 12:11:21.150 [ome.event.ItemCommandEvent] - Item 'HumidityPin' received command ON
2020-02-18 12:11:21.185 [nt.ItemStatePredictedEvent] - HumidityPin predicted to become ON
2020-02-18 12:11:21.190 [ome.event.ItemCommandEvent] - Item 'Pin23' received command ON
2020-02-18 12:11:21.194 [vent.ItemStateChangedEvent] - HumidityPin changed from NULL to ON
2020-02-18 12:11:21.207 [ome.event.ItemCommandEvent] - Item 'Pin18' received command ON
2020-02-18 12:11:21.213 [vent.ItemStateChangedEvent] - Pin23 changed from OFF to ON
2020-02-18 12:11:21.215 [vent.ItemStateChangedEvent] - Pin18 changed from OFF to ON
```

openhab.log
```
2020-02-18 12:10:42.601 [INFO ] [marthome.model.script.initialization] - Initializing settings
2020-02-18 12:11:01.474 [INFO ] [el.core.internal.ModelRepositoryImpl] - Refreshing model 'default.rules'
2020-02-18 12:11:01.874 [ERROR] [ntime.internal.engine.RuleEngineImpl] - Error during the execution of startup rule 'Initialization': null
2020-02-18 12:11:19.019 [INFO ] [marthome.model.script.initialization] - Initializing settings
2020-02-18 12:11:21.005 [INFO ] [marthome.model.script.initialization] - Sending 73 to MyTempProxy
2020-02-18 12:11:21.011 [INFO ] [marthome.model.script.initialization] - Sending 0 to PreviousTempReading
2020-02-18 12:11:21.037 [INFO ] [marthome.model.script.initialization] - Sending 36.9 to MyHumiProxy
2020-02-18 12:11:21.063 [INFO ] [marthome.model.script.initialization] - Sending 0 to PreviousHumiReading
2020-02-18 12:11:21.080 [INFO ] [smarthome.model.script.temp-setpoint] - Turning ON the heater
2020-02-18 12:11:21.085 [INFO ] [marthome.model.script.initialization] - Done initializing settings
2020-02-18 12:11:21.136 [INFO ] [smarthome.model.script.humi-setpoint] - Turning ON Dehumidify
```

Testing your commit now. Will update.

> Can you post events.log between the times where openhab.log shows "Initializing settings" and "Initializing settings" plus a second (just in case).

events.log:
```
2020-02-18 12:10:42.641 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.6
2020-02-18 12:10:42.659 [vent.ItemStateChangedEvent] - MyHumiProxy changed from NULL to 36.6
2020-02-18 12:10:42.684 [ome.event.ItemCommandEvent] - Item 'SystemType' received command EU
2020-02-18 12:10:42.692 [nt.ItemStatePredictedEvent] - SystemType predicted to become EU
2020-02-18 12:10:42.826 [ome.event.ItemCommandEvent] - Item 'TempUnit' received command F
2020-02-18 12:10:42.834 [nt.ItemStatePredictedEvent] - TempUnit predicted to become F
2020-02-18 12:10:44.642 [vent.ItemStateChangedEvent] - MyHumi changed from 36.6 to 36.7
2020-02-18 12:10:54.821 [vent.ItemStateChangedEvent] - MyHumi changed from 36.7 to 36.8
2020-02-18 12:10:59.312 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.8
2020-02-18 12:10:59.326 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:10:59.349 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.8
2020-02-18 12:10:59.354 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:10:59.367 [vent.ItemStateChangedEvent] - MyHumiProxy changed from 36.6 to 36.8
2020-02-18 12:10:59.373 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.8
2020-02-18 12:10:59.380 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.8
2020-02-18 12:10:59.408 [ome.event.ItemCommandEvent] - Item 'MainSwitch' received command ON
2020-02-18 12:10:59.656 [ome.event.ItemCommandEvent] - Item 'HeatingMode' received command ON
2020-02-18 12:10:59.693 [ome.event.ItemCommandEvent] - Item 'HotWaterMode' received command ON
2020-02-18 12:10:59.955 [ome.event.ItemCommandEvent] - Item 'HumidityMode' received command ON
2020-02-18 12:10:59.966 [ome.event.ItemCommandEvent] - Item 'Heating2' received command OFF
2020-02-18 12:10:59.981 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:10:59.989 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:11:00.028 [ome.event.ItemCommandEvent] - Item 'TempSetpoint' received command 90
2020-02-18 12:11:00.060 [ome.event.ItemCommandEvent] - Item 'TempSetpointF' received command 90
2020-02-18 12:11:00.090 [ome.event.ItemCommandEvent] - Item 'HumiSetpoint' received command 25
2020-02-18 12:11:00.107 [ome.event.ItemCommandEvent] - Item 'HeatingBoostTime' received command 10
2020-02-18 12:11:00.129 [ome.event.ItemCommandEvent] - Item 'HotWaterBoostTime' received command 10
2020-02-18 12:11:00.184 [ome.event.ItemCommandEvent] - Item 'HumiBoostTime' received command 10
2020-02-18 12:11:00.191 [ome.event.ItemCommandEvent] - Item 'PreviousTempReading' received command 73
2020-02-18 12:11:00.215 [ome.event.ItemCommandEvent] - Item 'PreviousHumiReading' received command 37.0
2020-02-18 12:11:00.223 [ome.event.ItemCommandEvent] - Item 'Heating2Time' received command 3
2020-02-18 12:11:00.236 [ome.event.ItemCommandEvent] - Item 'Heating2Delta' received command 2
2020-02-18 12:11:00.422 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_IP' received command 10.xx.xx.xx
2020-02-18 12:11:00.427 [nt.ItemStatePredictedEvent] - Network_WLAN_IP predicted to become 10.xx.xx.xx
2020-02-18 12:11:00.480 [ome.event.ItemCommandEvent] - Item 'Network_SSID' received command xxx
2020-02-18 12:11:00.497 [nt.ItemStatePredictedEvent] - Network_SSID predicted to become xxx
2020-02-18 12:11:00.598 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_INFO' received command 92
2020-02-18 12:11:00.625 [nt.ItemStatePredictedEvent] - Network_WLAN_INFO predicted to become 92
2020-02-18 12:11:00.630 [vent.ItemStateChangedEvent] - Network_WLAN_INFO changed from 86 to 92
2020-02-18 12:11:00.667 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_MAC' received command b8:27:eb:xx:xx:43
2020-02-18 12:11:00.684 [nt.ItemStatePredictedEvent] - Network_WLAN_MAC predicted to become b8:27:eb:xx:xx:43
2020-02-18 12:11:00.697 [vent.ItemStateChangedEvent] - Network_WLAN_MAC changed from NULL to b8:27:eb:xx:xx:43
2020-02-18 12:11:00.773 [ome.event.ItemCommandEvent] - Item 'System_CPU_TEMP' received command 122 °F
2020-02-18 12:11:00.788 [nt.ItemStatePredictedEvent] - System_CPU_TEMP predicted to become 122 °F
2020-02-18 12:11:01.859 [ome.event.ItemCommandEvent] - Item 'System_CPU_LOAD' received command 37
2020-02-18 12:11:01.866 [nt.ItemStatePredictedEvent] - System_CPU_LOAD predicted to become 37
2020-02-18 12:11:01.870 [vent.ItemStateChangedEvent] - System_CPU_LOAD changed from 49 to 37
2020-02-18 12:11:04.991 [vent.ItemStateChangedEvent] - MyHumi changed from 36.8 to 36.9
2020-02-18 12:11:19.044 [ome.event.ItemCommandEvent] - Item 'SystemType' received command EU
2020-02-18 12:11:19.049 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.9
2020-02-18 12:11:19.060 [nt.ItemStatePredictedEvent] - SystemType predicted to become EU
2020-02-18 12:11:19.069 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.9
2020-02-18 12:11:19.082 [ome.event.ItemCommandEvent] - Item 'TempUnit' received command F
2020-02-18 12:11:19.085 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:11:19.092 [vent.ItemStateChangedEvent] - MyHumiProxy changed from 36.8 to 36.9
2020-02-18 12:11:19.099 [nt.ItemStatePredictedEvent] - TempUnit predicted to become F
2020-02-18 12:11:19.105 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:11:19.126 [ome.event.ItemCommandEvent] - Item 'MainSwitch' received command ON
2020-02-18 12:11:19.145 [ome.event.ItemCommandEvent] - Item 'HeatingMode' received command ON
2020-02-18 12:11:19.164 [ome.event.ItemCommandEvent] - Item 'HotWaterMode' received command ON
2020-02-18 12:11:19.178 [ome.event.ItemCommandEvent] - Item 'HumidityMode' received command ON
2020-02-18 12:11:19.201 [ome.event.ItemCommandEvent] - Item 'Heating2' received command OFF
2020-02-18 12:11:19.232 [ome.event.ItemCommandEvent] - Item 'HumidityType' received command Dehumidify
2020-02-18 12:11:19.237 [nt.ItemStatePredictedEvent] - HumidityType predicted to become Dehumidify
2020-02-18 12:11:19.275 [ome.event.ItemCommandEvent] - Item 'TempSetpoint' received command 90
2020-02-18 12:11:19.306 [ome.event.ItemCommandEvent] - Item 'TempSetpointF' received command 90
2020-02-18 12:11:19.325 [ome.event.ItemCommandEvent] - Item 'HumiSetpoint' received command 25
2020-02-18 12:11:19.353 [ome.event.ItemCommandEvent] - Item 'HeatingBoostTime' received command 10
2020-02-18 12:11:19.368 [ome.event.ItemCommandEvent] - Item 'HotWaterBoostTime' received command 10
2020-02-18 12:11:19.379 [ome.event.ItemCommandEvent] - Item 'HumiBoostTime' received command 10
2020-02-18 12:11:19.406 [ome.event.ItemCommandEvent] - Item 'PreviousTempReading' received command 73
2020-02-18 12:11:19.413 [ome.event.ItemCommandEvent] - Item 'PreviousHumiReading' received command 37.0
2020-02-18 12:11:19.433 [ome.event.ItemCommandEvent] - Item 'Heating2Time' received command 3
2020-02-18 12:11:19.455 [ome.event.ItemCommandEvent] - Item 'Heating2Delta' received command 2
2020-02-18 12:11:19.503 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_IP' received command 10.xx.xx.xx
2020-02-18 12:11:19.512 [nt.ItemStatePredictedEvent] - Network_WLAN_IP predicted to become 10.xx.xx.xx
2020-02-18 12:11:19.552 [ome.event.ItemCommandEvent] - Item 'Network_SSID' received command xxx
2020-02-18 12:11:19.559 [nt.ItemStatePredictedEvent] - Network_SSID predicted to become xxx
2020-02-18 12:11:19.609 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_INFO' received command 90
2020-02-18 12:11:19.614 [nt.ItemStatePredictedEvent] - Network_WLAN_INFO predicted to become 90
2020-02-18 12:11:19.703 [vent.ItemStateChangedEvent] - Network_WLAN_INFO changed from 92 to 90
2020-02-18 12:11:19.745 [ome.event.ItemCommandEvent] - Item 'Network_WLAN_MAC' received command b8:27:eb:xx:xx:43
2020-02-18 12:11:19.750 [nt.ItemStatePredictedEvent] - Network_WLAN_MAC predicted to become b8:27:eb:xx:xx:43
2020-02-18 12:11:19.798 [ome.event.ItemCommandEvent] - Item 'System_CPU_TEMP' received command 122 °F
2020-02-18 12:11:19.802 [nt.ItemStatePredictedEvent] - System_CPU_TEMP predicted to become 122 °F
2020-02-18 12:11:20.859 [ome.event.ItemCommandEvent] - Item 'System_CPU_LOAD' received command 0
2020-02-18 12:11:20.865 [nt.ItemStatePredictedEvent] - System_CPU_LOAD predicted to become 0
2020-02-18 12:11:20.886 [vent.ItemStateChangedEvent] - System_CPU_LOAD changed from 37 to 0
2020-02-18 12:11:20.920 [ome.event.ItemCommandEvent] - Item 'System_Used_Space' received command 45 %
2020-02-18 12:11:20.928 [nt.ItemStatePredictedEvent] - System_Used_Space predicted to become 45 %
2020-02-18 12:11:21.018 [ome.event.ItemCommandEvent] - Item 'MyTempProxy' received command 73
2020-02-18 12:11:21.021 [nt.ItemStatePredictedEvent] - MyTempProxy predicted to become 73
2020-02-18 12:11:21.041 [ome.event.ItemCommandEvent] - Item 'PreviousTempReading' received command 0
2020-02-18 12:11:21.045 [vent.ItemStateChangedEvent] - MyTempProxy changed from NULL to 73
2020-02-18 12:11:21.048 [vent.ItemStateChangedEvent] - PreviousTempReading changed from 73 to 0
2020-02-18 12:11:21.065 [ome.event.ItemCommandEvent] - Item 'MyHumiProxy' received command 36.9
2020-02-18 12:11:21.070 [vent.ItemStateChangedEvent] - MyTempProxyF changed from NULL to 73
2020-02-18 12:11:21.078 [nt.ItemStatePredictedEvent] - MyHumiProxy predicted to become 36.9
2020-02-18 12:11:21.094 [ome.event.ItemCommandEvent] - Item 'PreviousHumiReading' received command 0
2020-02-18 12:11:21.098 [ome.event.ItemCommandEvent] - Item 'MyTempProxy' received command 73
2020-02-18 12:11:21.111 [nt.ItemStatePredictedEvent] - MyTempProxy predicted to become 73
2020-02-18 12:11:21.117 [ome.event.ItemCommandEvent] - Item 'HeatingPin' received command ON
2020-02-18 12:11:21.121 [vent.ItemStateChangedEvent] - PreviousHumiReading changed from 37.0 to 0
2020-02-18 12:11:21.139 [nt.ItemStatePredictedEvent] - HeatingPin predicted to become ON
2020-02-18 12:11:21.148 [vent.ItemStateChangedEvent] - HeatingPin changed from NULL to ON
2020-02-18 12:11:21.150 [ome.event.ItemCommandEvent] - Item 'HumidityPin' received command ON
2020-02-18 12:11:21.185 [nt.ItemStatePredictedEvent] - HumidityPin predicted to become ON
2020-02-18 12:11:21.190 [ome.event.ItemCommandEvent] - Item 'Pin23' received command ON
2020-02-18 12:11:21.194 [vent.ItemStateChangedEvent] - HumidityPin changed from NULL to ON
2020-02-18 12:11:21.207 [ome.event.ItemCommandEvent] - Item 'Pin18' received command ON
2020-02-18 12:11:21.213 [vent.ItemStateChangedEvent] - Pin23 changed from OFF to ON
2020-02-18 12:11:21.215 [vent.ItemStateChangedEvent] - Pin18 changed from OFF to ON
```

openhab.log
```
2020-02-18 12:10:42.601 [INFO ] [marthome.model.script.initialization] - Initializing settings
2020-02-18 12:11:01.474 [INFO ] [el.core.internal.ModelRepositoryImpl] - Refreshing model 'default.rules'
2020-02-18 12:11:01.874 [ERROR] [ntime.internal.engine.RuleEngineImpl] - Error during the execution of startup rule 'Initialization': null
2020-02-18 12:11:19.019 [INFO ] [marthome.model.script.initialization] - Initializing settings
2020-02-18 12:11:21.005 [INFO ] [marthome.model.script.initialization] - Sending 73 to MyTempProxy
2020-02-18 12:11:21.011 [INFO ] [marthome.model.script.initialization] - Sending 0 to PreviousTempReading
2020-02-18 12:11:21.037 [INFO ] [marthome.model.script.initialization] - Sending 36.9 to MyHumiProxy
2020-02-18 12:11:21.063 [INFO ] [marthome.model.script.initialization] - Sending 0 to PreviousHumiReading
2020-02-18 12:11:21.080 [INFO ] [smarthome.model.script.temp-setpoint] - Turning ON the heater
2020-02-18 12:11:21.085 [INFO ] [marthome.model.script.initialization] - Done initializing settings
2020-02-18 12:11:21.136 [INFO ] [smarthome.model.script.humi-setpoint] - Turning ON Dehumidify
```

Testing your commit now. Will update.

--- a/home/pi/scripts/default.rules
+++ b/home/pi/scripts/default.rules
@@ -367,6 +367,15 @@ then
     }
   ]
 
+  // All the modes are set but they won't change to trigger the Mode rule below
+  // so call the lambdas to cause the devices to turn on/off as appropriate.
+  heating.apply(initLogName, TIMERS, heating_ctrl)
+  if(SystemType.state == "EU") {
+    humidity.apply(initLogName, TIMERS, humidity_ctrl)
+    hotwater.apply(initLogName, TIMERS, hotwater_ctrl)
+  }
+  else cooling.apply(initLogName, TIMERS, cooling_ctrl)
+
   logInfo(initLogName, "Done initializing settings")
 end
 
@@ -868,4 +877,3 @@ then
     logWARN("wanip", "Unable to get WAN IP")
   }
 end
-

--- a/home/pi/scripts/default.rules
+++ b/home/pi/scripts/default.rules
@@ -354,19 +354,6 @@ then
     m.sendCommand(resetVal) // Changing from Boost will reset the RemBoostTime Item and cancel the timers
   ]
 
-  // Send out the change to the current readings to kick off the selected mode.
-  // The sensor should have updated at least once while this Rule was running.
-  Sensors.members.forEach[ s |
-    if(s.state instanceof UnDefType) logError(logName, s.name + " doesn't have a value!")
-    else {
-      val type = s.name.replace("My", "")
-      logInfo(initLogName, "Sending " + s.state + " to " + s.name + "Proxy")
-      sendCommand(s.name+"Proxy", s.state.toString)
-      logInfo(initLogName, "Sending 0 to Previous"+type+"Reading")
-      sendCommand("Previous"+type+"Reading", "0")
-    }
-  ]
-
   // All the modes are set but they won't change to trigger the Mode rule below
   // so call the lambdas to cause the devices to turn on/off as appropriate.
   heating.apply(initLogName, TIMERS, heating_ctrl)

